EnglishFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/77918
+bscscan.com#?#.text-center:contains(Ad)
```

Comment:
Why not `bscscan.com##.showcase-banner-spacing`?

Reply:
What a magician! Where did you even find it?

Reply:
In their html code.